macros/src/main/scala/gestalt/macros/Optional.scala

```diff
@@ -28,11 +28,14 @@ final class Optional[+A >: Null](val value: A) extends AnyVal {
         Select(tempIdent, "value")
     }
 
-    q"""
+    {
+      import scala.gestalt.options.unsafe
```

Comment:
After the refactoring, it doesn't make sense to protect `NewInstance` by `Unsafe` any more. We can write `new _root_.macros.Optional`, and it's hygienic.